### PR TITLE
Fix the changelog

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 1]
```diff
@@ -29,17 +29,21 @@ Information
    ``service: alembic`` by ``service: alembic-advance``.
 
 4. Some little changes in the application build:
-   - We no longer use custom build commands to build the docker images, we simply use the docker-compose
-     build command, with two additional environment variables:
-     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+
+   - In ``./build`` script, we no longer call ``docker build`` command with custom parameters, but call
+     ``docker-compose build`` command with wo additional environment variables:
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:34]: wo -> two

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 2]
```diff
@@ -29,17 +29,21 @@ Information
    ``service: alembic`` by ``service: alembic-advance``.
 
 4. Some little changes in the application build:
-   - We no longer use custom build commands to build the docker images, we simply use the docker-compose
-     build command, with two additional environment variables:
-     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+
+   - In ``./build`` script, we no longer call ``docker build`` command with custom parameters, but call
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:33]: In -> In the
call -> call the

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 3]
```diff
@@ -29,17 +29,21 @@ Information
...
      * ``GIT_HASH``: the git hash of the current version.
+
      The build specification has been moved to the ``docker-compose`` files.
    - The ``--geoportal`` and ``--config`` arguments have been replaced by ``--service=geoportal`` and
      ``--service=config`` respectively.
-   - The image used by the ``docker-compose`` will be pulled from the Docker hub.
-   - New argument ``--fast-reload`` that will recreate all containers and restart them except the redis
-     containers to keep the cache.
-   - New argument ``--dry-run``.
+   - The external image used in the composition are pulled from the Docker hub on each build.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:42]: images -> images

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 4]
```diff
@@ -29,17 +29,21 @@ Information
...
+   - ``./build`` script support a new argument ``--fast-reload`` that will recreate all containers except
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:43]: support -> supports

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 5]
```diff
@@ -29,17 +29,21 @@ Information
...
+     Redis containers to keep the cache.
+   - ``./build`` script support a new argument ``--dry-run`` which only display the ``docker`` and
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:45]: support -> supports
display -> displays

**Review by jwkaltz — APPROVED** [review]

**Review by arnaud-morvan — APPROVED** [review]

**Comment by sbrunner** [comment]
Thanks for the reviews :-)

**Pull request merged**